src/libxtp/calculators/eanalyze.cc

```diff
@@ -36,6 +36,13 @@ void EAnalyze::Initialize(const tools::Property &user_options) {
   }
 
   std::string statestrings = options.get(".states").as<std::string>();
+  std::size_t start_bracket = statestrings.find('[');
```

Comment:
wait I wanted the [] only for the definition of choices not for the values themselves

i.e. `<levels choice=[a,b,c]>a,b,c</levels>`

not
`<levels choice=[a,b,c]>[a,b,c]</levels>`

Reply:
my unclear communication I am sorry.